bears/r/RLintBear.py

```diff
@@ -24,9 +24,10 @@ class RLintBear:
     LANGUAGES = {'R'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement('lintr'),
+    REQUIREMENTS = {RscriptRequirement('lintr',
+                                       version='>=1.0.2'),
                     DistributionRequirement(apt_get='r-base',
-                                            version='>=3.1.1')}
```

Comment:
Please leave this version unmodified.  This is the version of `r-base` which is needed.